angular/src/app/components/modal-questionnaire-viewer/modal-questionnaire-viewer.component.styl

```diff
@@ -19,1403 +19,1470 @@
 *    by accessing https://rapid.apl.uw.edu/rapp/ and retreiving the code as the code is not minimized.
 */
 
-::ng-deep p.questionnaireTitle {
```

Comment:
I dropped that selector entirely in my PR. If it is needed, we can add back but if it is 
for linting/other, we don't need it because it wont exist anymore after we merge in my PR. ;)

Reply:
Sorry yes, I reverted these changes so they're not included in the PR.